scripts/do-release.sh

```diff
@@ -101,17 +102,12 @@ function update_go_mod() {
 }
 
 function update_dependencies() {
-    local msg="$1"
-    shift
-
     clone_and_create_branch "update-dependencies-${release['branch']:-devel}"
 
-    for dependency; do
-        update_go_mod "$dependency"
-    done
+    [ ! -f "projects/${project}/go.mod" ] || update_go_mod "$dependency"
```

Comment:
`$dependency` is no longer defined plus `update_go_mod` no longer takes an arg.

```suggestion
    [ ! -f "projects/${project}/go.mod" ] || update_go_mod
```